### DISCUSSION TITLE
Tutorial on using the box-shadow support for themes

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#47] [Topic Ideas]
Gutenberg 14.9 introduces a new "shadow" preset for theme.json: https://github.com/WordPress/gutenberg/pull/46813

This allows devs to create a set of custom box-shadow presets for use in their themes, which can be referenced in `theme.json` or templates.  We need a tutorial that covers how to implement this. 

Currently, we are still waiting on UI integration in the form of a component on the user's end.  Do y'all think it's worth waiting for the UI pieces to land before the tutorial?  

I'm up for writing this.

**Comment by bph** [comment]
It would be great to have a tutorial now, so theme developers can test it and apply it to different scenarios. Their feedback would help refine the functionality and inform UI design.

**Comment by bph** [comment]
Published: [Using the box shadow feature for themes](https://developer.wordpress.org/news/2023/01/using-the-box-shadow-feature-for-themes/)
